proposals/simple-lambda-parameters-with-modifiers.md

```diff
@@ -62,60 +57,20 @@ anonymous_function_parameter_modifier
-
-## Answered LDM questions
-
-### Allowing attributes or default parameter values
-
-Both seem viable, and may be worth it if we're doing the rest of this work.  With this formalization, we would likely instead say that:
-
-```diff
-explicit_anonymous_function_parameter
--    : attributes anonymous_function_parameter_modifier? type identifier default_argument?
-+    : attributes anonymous_function_parameter_modifier? type? identifier default_argument?
-    ;
-```
+In a `lambda_parameter_list` all `lambda_parameter` elements must either have a `type`
+present or not have a `type` present.  The former is considered to be an "explicitly
+typed parameter list", while the latter is considered to be an "implicitly typed
+parameter list".
 
-Note: This grammar is technically ambiguous with implicit_anonymous_function_parameter.  We'd need
-to clarify that at least one of `attributes, modifier, type or default-argument` would have to be provided.
-We'd also need a rule that all parameters in the list would have to supply a type, or eschew a type.
+Parameters in an implicitly typed parameter list cannot have a `default_argument`.  They
+can have an `attribute_list`.
 
-We would also update the semantic specification to say:
+The following change is required to [anonymous function conversions](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-12.0/lambda-method-group-defaults.md#detailed-design):
 
 ```diff
-- The parameters of an anonymous function in the form of a lambda_expression can 
-- be explicitly or implicitly typed. In an explicitly typed parameter list, the type
-- of each parameter is explicitly stated. In an implicitly typed parameter list, 
-- the types of the parameters are inferred from the context in which the anonymous 
--function occurs—specifically, when the anonymous function is converted to a compatible
-- delegate type or expression tree type, that type provides the parameter types (§10.7).
-+ The parameters of an anonymous function in the form of a lambda_expression can 
-+ be explicitly or implicitly typed. In an `anonymous_function_signature` whose 
-+ parameters have a provided `type`, the type of each parameter is explicitly stated. 
-+ In all other signatures the types of the parameters are inferred from the context 
-+ in which the anonymous function occurs—specifically, when the anonymous function 
-+ is converted to a compatible delegate type or expression tree type, that type 
-+ provides the parameter types (§10.7).
+[...]
+- If F has an explicitly typed parameter list, each parameter in D has the same type and
+- modifiers as the corresponding parameter in F ignoring params modifiers and default values.
++ If F has an explicitly or implicitly typed parameter list, each parameter in D has the same type and
++ modifiers as the corresponding parameter in F ignoring params modifiers and default values.
```

Comment:
Consider using `diff` for code blocks only, such as grammar changes. For changes to formatted sections, let's use the existing formatting plus **bold** for added text and ~~strike-through~~ for deleted text. Otherwise, the modified blocks are often rendered as unformatted and completed replaced.

Suggestion:
```suggestion
> If F has an explicitly **or implicitly typed parameter list**, each parameter in D has the same type and
> modifiers as the corresponding parameter in F ignoring params modifiers and default values.
```